snippets/product-main-info.liquid

```diff
@@ -131,12 +142,12 @@
     <div class="product-form__info-item product-form__info-price">
       {%comment%}<span class="product-form__info-title text--strong">{{ 'product.form.price' | t }}:</span>{%endcomment%}
 
-      <div class="product-form__info-content" role="region" aria-live="polite">
+      <div class="product-form__info-content display_offer_image" role="region" aria-live="polite">
         <div class="price-list actual-product-price">
           {%- if selected_variant.compare_at_price > selected_variant.price -%}
           <span class="price price--highlight">
             <span class="visually-hidden">{{ 'product.general.sale_price' | t }}</span>
-            {{- selected_variant.price | money -}}  &nbsp;<small style="color:#777; font-size:18px;display:inline-block;margin-bottom:11px;">MRP </small>
+            <small style="color:#9a7058; font-size:23px;display:inline-block;margin-bottom:2px;">MRP: </small> &nbsp; {{- selected_variant.price | money -}}  
```

Comment:
CSS in style tags must be avoided. @AvishekPrasad

Reply:
Sir It was already present so I have'nt changed it , just modified the css that was required